components/itinerary/Itinerary.js

```diff
@@ -180,6 +187,7 @@ const Itinerary = ({ isMobile }: Props) => (
                 <DropdownWrapper>
                   <Dropdown onChange={changeDropdownValue} options={dropdownData} />
                 </DropdownWrapper>
+                <div />
```

Comment:
?

Reply:
That div is used for spacing. It's used to replace this part:

```
const DropdownGroup = styled.div`
  ...
  grid-template-columns: 1fr 26% 1fr;
```

What I did is make `DropdownGroup` have 3 children, the first and last (that div would be the last) will expand, while the second one will have fixed width of 26%.

It's the cleanest solution I could find, but if you have some suggestions I can change it.

Reply:
what about havin the first child as 37%?